src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  background-color: $color-white;
```

Comment:
Ensure that the variable `$color-white` is defined in your SCSS files. If not, this will cause a compilation error.